test/commands.test.js

```diff
@@ -621,7 +622,12 @@ test('[commands.operations.getMasterConfig] success', function(assert) {
     }
   });
 
-  context.oldParameters = { old: 'stale' };
+  AWS.stub('KMS', 'decrypt', function(params, data) {
+    return 'data';
+  });
+
+  context.oldParameters = { old: 'secure:staleelats' };
+  context.overrides = { templateOptions : { AWS_DEFAULT_REGION : 'us-east-1'}};
```

Comment:
Does this PR introduce a dependency on an `$AWS_DEFAULT_REGION` env var that wasn't there before? Or is it just this test that needed to be changed?